shapely/geometry/base.py

```diff
@@ -77,26 +76,32 @@ def _maybe_unpack(result):
 
 
 class CAP_STYLE:
+    """Buffer cap styles."""
+
     round = BufferCapStyle.round
     flat = BufferCapStyle.flat
     square = BufferCapStyle.square
 
 
 class JOIN_STYLE:
+    """Buffer join styles."""
+
     round = BufferJoinStyle.round
     mitre = BufferJoinStyle.mitre
     bevel = BufferJoinStyle.bevel
 
 
 class BaseGeometry(shapely.Geometry):
-    """
-    Provides GEOS spatial predicates and topological operations.
-
-    """
+    """Provides GEOS spatial predicates and topological operations."""
 
     __slots__ = []
 
     def __new__(self):
+        """Directly calling the base class 'BaseGeometry()' is deprecated.
+
+        This will raise an error in the future. To create an empty geometry,
+        use one of the subclasses instead, for example 'GeometryCollection()'
+        """
```

Comment:
Does ruff / pydocstyle require this docstring to be present? Because this is just superfluous (repeating the warning below, and this is not actually included in the documentation)

Reply:
I suppose this is https://docs.astral.sh/ruff/rules/undocumented-magic-method/